core/src/main/scala/org/bitcoins/core/protocol/transaction/TransactionConstants.scala

```diff
@@ -11,6 +11,7 @@ trait TransactionConstants {
   lazy val validLockVersion = Int32(2)
   lazy val lockTime = UInt32.zero
   lazy val sequence = UInt32(4294967295L)
+  lazy val disableRBFSequence = UInt32(4294967294L)
```

Comment:
nit: maybe make this `sequence - 1` so it is clearer to see that it is `MAX - 1`

Reply:
This enables RBF: https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki#summary


>Explicit signaling: A transaction is considered to have opted in to allowing replacement of itself if any of its inputs have an nSequence number less than (0xffffffff - 1).